addons/mail/static/src/messaging/entity/chat_window/chat_window.js

```diff
@@ -312,8 +312,8 @@ function ChatWindowFactory({ Entity }) {
             if (!this.env.entities.Device || !this.env.entities.Discuss) {
```

Comment:
Ok, also the following condition `!device || !discuss` is useless the same way.

Reply:
Don't think so, because this method could be called even though `messaging`, `device`, or `discuss` haven't yet been initialized & registered in `env` yet.

Reply:
`|| !this.env.messaging.isInitialized`